apps/src/applab/applab.js

```diff
@@ -782,7 +784,14 @@ Applab.init = function (config) {
         });
       }
     });
-
+  // Only send PROJECT_ACTIVITY event for students or non-authenticated users,
+  // AND only if they're on a project level.
+  if (
+    getStore().getState().currentUser?.userType !== 'teacher' &&
```

Comment:
```suggestion
    !getStore().getState().currentUser?.isTeacher &&
```

Reply:
We could create a separate function for this, allowing us to import and reuse it across the app:
```js
export const reportTeacherProjectActivityEvent = () => {
  if (!getStore().getState().currentUser?.isTeacher) {
    analyticsReporter.sendEvent(EVENTS.PROJECT_ACTIVITY, {}, PLATFORMS.BOTH);
  }
};
```

```js
import {reportTeacherProjectActivityEvent} from '...';

if (config.level.isProjectLevel) reportTeacherProjectActivityEvent();
```

Reply:
I've removed the conditional teacher check all together as I don't believe it's necessary.